recipe/meta.yaml

```diff
@@ -12,16 +12,16 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
-    - python >=3.6
-    - pbr
+    - python
+    - pbr ==5.6.0
```

Comment:
I believe you don't actually need the `==` but I also think it doesn't really matter.

Reply:
I would advice to unpin it.